utils/paste.test.tsx

```diff
@@ -68,7 +68,7 @@ describe('Paste.formatGithubCodePaste', () => {
         const message = "```\n// a javascript codeblock example\nif (1 > 0) {\n  return 'condition is true';\n}\n```";
         const codeBlock = "```\n// a javascript codeblock example\nif (1 > 0) {\n  return 'condition is true';\n}\n```";
 
-        const {formattedMessage, formattedCodeBlock} = formatGithubCodePaste(0, '', clipboardData);
+        const {formattedMessage, formattedCodeBlock} = formatGithubCodePaste(0, '', clipboardData); // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
Same with these.

Reply:
I filed a ticket to fix these. The tests weren't written correctly, so I added the ignore comments so that the argument handling doesn't need to be re-added when the tests are fixed.

Reply:
Once https://github.com/mattermost/mattermost-webapp/pull/5559 is merged, these can be removed. I'll do that in this PR if that one makes it in first.